assembly.py

```diff
@@ -489,6 +563,15 @@ def parser_order_and_orient(parser=argparse.ArgumentParser()):
                 but were not chosen for the final output.""",
         default=None
     )
+
+    parser.add_argument('--nGenomeSegments', dest='n_genome_segments', type=int, default=0,
+                        help="""Number of genome segments.  If 0 (the default), the `inReference` parameter is treated as one genome.
+                        If positive, the `inReference` parameter is treated as a list of genomes of nGenomeSegments each.""")
+
+    parser.add_argument('--threads', type=int, default=0, help='Number of threads to use (0 for all cpus)')
```

Comment:
can you remove the `--threads` argument and add `threads to the `util.cmd.common_args` invocation a few lines below instead?